doc/nrf/includes/vsc_build_and_run_nrf53.txt

```diff
@@ -0,0 +1,14 @@
+1. Open Visual Studio code.
```

Comment:
Also, this file is in `.txt` format. Should it be an `.rst`?

Reply:
`.txt` is okay since it's only for holding the include, not to be built itsef.